apple/xcode/maplibre-rs.xcodeproj/project.pbxproj

```diff
@@ -804,7 +804,7 @@
 					"$(inherited)",
 					"@executable_path/../Frameworks",
 				);
-				MACOSX_DEPLOYMENT_TARGET = 12.3;
+				MACOSX_DEPLOYMENT_TARGET = 10.9;
```

Comment:
Ideally this should be done in a separate task, but let's keep it like that!

Reply:
I am sorry this was a mistake, is it ok if I make a new commit and then we squash this pull request before merging? Same goes for the `misc.xml` file. I have inadvertently commited these two files...

I want to do some research, before changing the OSX deployment target, to select the most appropriate one.